src/equations/2d/linear_scalar_advection.jl

```diff
@@ -24,18 +24,27 @@ get_name(::LinearScalarAdvectionEquation2D) = "LinearScalarAdvectionEquation2D"
 varnames_cons(::LinearScalarAdvectionEquation2D) = SVector("scalar")
 varnames_prim(::LinearScalarAdvectionEquation2D) = SVector("scalar")
 
+function x_trans_periodic(x)
+  #Fixed domain_length and center.
+  domain_length = [2,2]
+  center = [0,0]
```

Comment:
Thanks for working on this! I think it would be better to make `domain_length` and `center` optional arguments with default value. Moreover, it would be nice to make some benchmarks of this, e.g. using BenchmarkTools.jl. For example, I expect to see some performance gain (less allocations, faster execution) by using `SVector`s instead of plain `Vector`s for these parameters.

Reply:
Thanks for your feedback. I´ll add optional arguments for thedomain_length and center variable and change to SVectors.